### DISCUSSION TITLE
How to realize auto scale for y axis?

### DISCUSSION THREAD

**Discussion by makaronis** [patrykandpatrick/vico#323] [Questions] [answered]
How can i realize this scale feature? I need to know what first and last entry is currently visible on screen

**Comment by patrickmichalik** [comment, marked as answer]
Hello! At the moment, this feature isn’t available. As I understand, you’re looking to implement it yourself. You can determine what entries are visible by using the width of the `Chart` bounds, the scroll amount, and `HorizontalDimensions`. `HorizontalAxis`, for example, calculates what _x_ values are visible—see [here](https://github.com/patrykandpatrick/vico/blob/master/vico/core/src/main/java/com/patrykandpatrick/vico/core/axis/horizontal/HorizontalAxis.kt#L105-L107). The intersection of this and the _x_ values in the data set is what you’re looking for.

**Reply by makaronis** [reply]
Thanks for that information sir

**Reply by patrickmichalik** [reply]
Sure thing!
